test/web-auth/web-auth.test.js

```diff
@@ -1754,7 +1754,8 @@ describe('auth0.WebAuth', function() {
         clientID: '...',
         redirectUri: 'http://page.com/callback',
         responseType: 'code',
-        _sendTelemetry: false
+        _sendTelemetry: false,
+        nonce: 'the-nonce'
```

Comment:
You shouldn't be setting the nonce value yourself. The PR description clearly states that what was broken is "state/nonce were not being auto generated". Please DO check that the nonce is present but don't manually set it.

Reply:
I think this line is already doing some of that https://github.com/auth0/auth0.js/blob/f6e2227bfe921e30fae73a31b033d12255441d08/test/web-auth/web-auth.test.js#L27

Reply:
you're right. I switched to id_token since it's the only case where it generates the nonce.